sdks/python/apache_beam/runners/worker/statesampler_fast.pyx

```diff
@@ -101,6 +103,13 @@ cdef class StateSampler(object):
     pythread.PyThread_acquire_lock(self.lock, pythread.WAIT_LOCK)
     self.scoped_states_by_index = [unknown_state]
     pythread.PyThread_release_lock(self.lock)
+    self.element_tracker = DataflowElementExecutionTracker()
+    # TODO: remove experimental flag once time_counter release
+    experiments = RuntimeValueProvider.get_value('experiments', str, '')
```

Comment:
You can use experiments [like this](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/worker/operations.py#L292). (I fixed my earlier bug :))

Reply:
Changed in latest commit. Thanks for fix!